src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { PropsWithChildren } from 'react';
+import * as S from './Stepper.style';
+
+export type StepperProps = PropsWithChildren<any>;
+
+const Stepper: React.FC<StepperProps> = ({ children, ...rest }: StepperProps) => (
+	<S.Stepper>
+		{React.Children.map(children, (child, index) => [
+			<div className="line" />,
+			React.cloneElement(child, { index: index + 1, ...rest }),
```

Comment:
You don't need to spread Stepper props here, spread into `<S.Stepper {...rest}>`